web/react/dispatcher/event_helpers.jsx

```diff
@@ -148,3 +148,8 @@ export function emitClearSuggestions(suggestionId) {
         id: suggestionId
     });
 }
+
+// convenience method to dispatch an event in JS' next event cycle
+export function defer(fn) {
```

Comment:
This function would be better in utils.jsx. This file is for functions that create events for the dispacher. The "action creators" in this flux diagram: https://facebook.github.io/react/img/blog/flux-diagram.png  We are going to change the name to "action creators" soon so it makes more sense.